galactic/distribution.yaml

```
@@ -535,6 +535,21 @@ repositories:
       url: https://github.com/eclipse-cyclonedds/cyclonedds.git
       version: iceoryx
     status: maintained
+  data_buffer:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/data_buffer.git
+      version: master
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/OUXT-Polaris/data_buffer-release.git
+      version: 0.0.1-1
+    source:
+      type: git
+      url: https://github.com/OUXT-Polaris/data_buffer-release.git
```

This should the the upstream repository

```suggestion
      url: https://github.com/OUXT-Polaris/data_buffer.git
```